config/config.js

```diff
@@ -1,26 +1,38 @@
 module.exports = {
     development: {
-        username: "gorupin",
+        username: "groupin",
         password: "Passwordm0reSeCret",
-        database: "groupin",
+        database: "groupin_development",
         host: "127.0.0.1",
         port : "3306",
-        dialect: "mysql"
+        dialect: "mysql",
+        define: {
+          charset: 'utf8',
```

Comment:
change this to utf8mb4 and the right collate, do this for all configuration envs